src/extension/background-script/actions/nostr/signEventOrPrompt.ts

```diff
@@ -17,15 +19,44 @@ const signEventOrPrompt = async (message: MessageSignEvent) => {
     };
   }
 
+  const allowance = await db.allowances.get({
+    host: message.origin.host,
+  });
+
+  if (!allowance?.id) {
+    return { error: "Could not find an allowance for this host" };
+  }
+
+  const findPermission = await db.permissions.get({
+    host: message.origin.host,
+    method: PermissionMethodNostr["NOSTR_SIGNMESSAGE"],
+  });
+
+  const hasPermission = !!findPermission?.enabled;
```

Comment:
Just to repeat again:
The check for "enabled" is _actually_ not needed.
Either the permission is saved to DB (enabled was set to "true" in the promptReponse once)
or the permission does not exist in the DB.

See this comment again for clarification and why this check is still implemented:
https://github.com/getAlby/lightning-browser-extension/pull/1724#discussion_r1017642768

Reply:
True, I just did that for _the future_. Can remove.